src/styles/globals.css

```diff
@@ -97,6 +98,10 @@ textarea {
 .dark {
   --maxPageWidth: 1024px;
   --contentPadding: var(--spacings-xLarge);
+  --labelWidth: 5rem;
```

Comment:
Is this the correct place for these variables?

Reply:
Are you thinking of globals.css in general or that specific class?

Reply:
`globals.css`. For me, these variables look more like local variables rather than global variables accessible everywhere. Can't see that these variables are needed outside the details page? (Haven't hunted down where they are used, so it might be)

Reply:
The reason why I chose .dark/.light is that we have other global variables defined here. Could move them all to `:root` or similar if that's better. 

These variables are needed by both the `Row` and `DecorationLine` components. Instead of defining them twice I thought it would be nice to have them in one place and I therefore went with the globals.css.

Can you think of a better place for them or another approach to this?

Reply:
^ I didn't refresh and didn't see your answer before posting 😄

Reply:
Personally I would have placed the variables in their common parent, which looks to be `.rowContainer`, if I'm looking at this correctly.